### DISCUSSION TITLE
How to build this for Ubuntu 20.04 aarch64?

### DISCUSSION THREAD

**Discussion by Danrancan** [Z-Hub/Z-Push#18] [Q&A] [answered]
I'm running Ubuntu 20.04 server on a raspberry pi. But there are no build instructions for this repository. Can someone please provide instructions on how to build this from source for Ubuntu 20.04 Raspi?

**Comment by matidau** [comment]
Heya,

This doesn't need to be built or compiled as such, just copy the source to the correct directory for your system, and configure it.

Depending on what backends you will be using, there are different places to get help on this.

Which backends were you planning on setting this up with?

**Comment by matidau** [comment, marked as answer]
Also if you look under the wiki, there is a copy of the Kopano Z-Push documentation and links to a few different resources. A lot of this does need a refresh but might give you something to start with immediately.

**Comment by matidau** [comment]
I've updated the Wiki with some very general instructions to get the latest release and to clone from git https://github.com/Z-Hub/Z-Push/wiki/Installation#z-push-on-ubuntu-2204 

These will also work for Ubuntu 20.04 as long as you have PHP 7.4 or greater available.

I've also opened up the Wiki so all Github users can contribute. If you have any updates to the documentation please feel free to add or edit what is there.
